CONTRIBUTING.md

```diff
@@ -47,3 +48,7 @@ Before submitting your pull request, please make sure that the following conditi
 * Your new code **passes all existing and new tests** through running `npm run test`.
 * Your PR **excludes distribution files** in `dist/**`.
 * You appended yourself to the **list of contributors** in the [NOTICE](./NOTICE) file.
+
+Please note that if a pull request is rather complicated, i.e. touches lots of internals, or became stale, it is not uncommon that a core contributor performs the final integration to get it done in good conscience while naming you as a co-author.
```

Comment:
I think it'd be worth duplicating the pull request tips in the pull request template itself. Something like:

```
<!--
 Thanks for submitting a pull request to AssemblyScript!
 Please follow the contributing guidelines linked below to get off to a good start.

Here are a few tips when opening a pull request:

* If this is not a small fix, Consider creating an issue to discuss first, and later link to it from the PR.

* If a pull request is rather complicated, i.e. touches lots of internals, or became stale, it is not uncommon that a core contributor performs the final integration to get it done in good conscience while naming you as a co-author.

Thank you for the pull request!
-->
```

Just because, I know personally I wouldn't stop, and then go look for the contributing docs if I saw this :joy: What do you think? :smile:

Reply:
Do you think adding a checkbox to tick

- [ ] I've read the contributing guidelines

would have a similar effect? Asking because I found myself being turned off from over-engineered issue templates in the past, so I am a bit biased towards keeping these short :)

Reply:
Yeah I think a checkbox works :smile: 

And ahhh, makes sense, if you want to keep it short, definitely let's do a checkbox :)